user-contrib/Ltac2/tac2core.ml

```diff
@@ -17,6 +17,28 @@ open Tac2expr
 open Tac2entries.Pltac
 open Proofview.Notations
 
+let constr_flags () =
```

Comment:
Why's this a function?

Reply:
Probably historical reasons, it used to have a field that was depending on a global flag. Note that this snippet has not been introduced by this PR, the code was just moved a bit before.

Reply:
Fixed.